packages/venia-ui/lib/components/Header/cartTrigger.css

```diff
@@ -1,9 +1,29 @@
-.root {
+.root_desktop {
```

Comment:
Generally, we use the `.root` classname for the component's root element. Here, the root is a fragment, so we wouldn't really have a root.

We also generally use the `x_y` format to indicate that `y` is one state of `x`. Here, `root_desktop` and `root_mobile` are sibling elements, so we should probably not use that format here. Further, `desktop` and `mobile` don't really get at the heart of the distinction here, so perhaps `trigger` (desktop, triggers a dropdown) and `link` (mobile, links to a new page) would distinguish the two elements better.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2494/commits/29cb5bef7ad1a0376bd8d7b418eadbb7dac3a7a7.